packages/TelegramClientTests-Core.package/TCTCTestCore.class/instance/receiveOrderOfEventSatisfying..st

```diff
@@ -0,0 +1,9 @@
+event log querying
+receiveOrderOfEventSatisfying: aBlock
+
+	| position |
+	
+	position := self eventLog findFirst: aBlock.
+	self assert: (position = 0) not.
```

Comment:
You can use `self deny:` instead of negating. (Do ot use `self deny:equals:` though because thats not available in Squeak 5.2.

Reply:
`Deny` unfortunately only works if the current object is an instance of `TestCase`, because TestCase defines all these extensions of `assert`...